### DISCUSSION TITLE
[Datatable] sort column desc first

### DISCUSSION THREAD

**Discussion by Rakasch** [primefaces/community#529] [PrimeVue]
Currently a Column is always sorted asc on first click, and desc on second click.

I would like to have more sorting configurations for the columns:
- Sort the column asc on first click, desc on second click (current default).
- Sort the column desc on first click, asc on second click.
- Sort the column asc only
- Sort the column desc only

**Comment by tugcekucukoglu** [comment]
- Sort the column desc on the first click, asc on the second click: adding `:defaultSortOrder="-1"` prop can achieve this.

For others, we can wait for the community demand.

**Reply by Rakasch** [reply]
defaultSortOrder is a datatable prop. It will change the default order for **ALL** columns. 
How would I configure it separately for a single column?
It didn't do anything, when I added defaultSortOrder="-1" to a column.
eg. I would want a "name" column to sort asc first, and a "date" column to sort desc first.
